### PR TITLE
Phase pistol

### PR DESCRIPTION
**Pull request by Anewbe**
5 Commits because mapping, changelog, and I'm a dingus.
- Replaces the Hunting Rifle cabinets with a Pulse Pistol cabinet. Pulse Pistols serve the same purpose, but they won't IB your buddies if you misclick.
- Drops some of the environmental resistances on Explorer gear. Bomb and Rad are down to 35, from 50. Bio is down to 75, from 100. Might want to take a quick pause before running into the Plague Ship, now.

### CODE REVIEW

**Comment by elgeonmb** [comment]
thank you

**Comment by Leshana** [comment]
That's good, 100% bio sounds like too much.

**Comment by PrismaticGynoid** [comment]
If this phase pistol uses weapon cells, will there be extras provided in the lockers, or will the explorers need to run back to the outpost to charge every time they run out?

**Comment by Anewbe** [comment]
One extra cell per gun is in the lockers.

**Comment by PrismaticGynoid** [comment]
8 shots per cell, right?

**Comment by Anewbe** [comment]
Correct.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -782,7 +782,7 @@ obj/item/clothing/suit/storage/toggle/peacoat
 	hooded = TRUE
 	hoodtype = /obj/item/clothing/head/hood/explorer
 	siemens_coefficient = 0.9
-	armor = list(melee = 30, bullet = 20, laser = 20, energy = 20, bomb = 50, bio = 100, rad = 50) // Inferior to sec vests in bullet/laser but better for environmental protection.
+	armor = list(melee = 30, bullet = 20, laser = 20, energy = 20, bomb = 35, bio = 75, rad = 35) // Inferior to sec vests in bullet/laser but better for environmental protection.
```
Comment (mistyLuminescence) [inline code/modules/clothing/suits/miscellaneous.dm]: 👀 lowering the bomb resistance... #4556

**Pull request merged**